static/include/micro-rdk.md

```diff
@@ -10,10 +10,12 @@ The only microcontroller the micro-RDK currently supports is the [ESP32](https:/
 
 [Client API](/program/apis/) usage with the Micro-RDK is currently limited to the following supported {{< glossary_tooltip term_id="resource" text="resources" >}}:
 
-- [Motor](/components/motor/)
-- [Encoder](/components/encoder/)
-- [Base](/components/base/)
-- [GPIO Pin](/components/board/#gpiopin-api)
```

Comment:
removed these as the parts of API that are supported are now covered on board page